src/components/molecules/SaveResourcesToFileFolderModal/SaveResourcesToFileFolderModal.tsx

```diff
@@ -0,0 +1,357 @@
+import React, {useCallback, useEffect, useMemo, useState} from 'react';
+
+import {Button, Input, Modal, Select} from 'antd';
+
+import {FolderAddOutlined} from '@ant-design/icons';
+
+import fs from 'fs';
+import micromatch from 'micromatch';
+import path from 'path';
+import styled from 'styled-components';
+import util from 'util';
+import {stringify} from 'yaml';
+
+import {ROOT_FILE_ENTRY, YAML_DOCUMENT_DELIMITER} from '@constants/constants';
+
+import {AlertEnum} from '@models/alert';
+
+import {useAppDispatch, useAppSelector} from '@redux/hooks';
+import {setAlert} from '@redux/reducers/alert';
+import {uncheckAllResourceIds} from '@redux/reducers/main';
+import {closeSaveResourcesToFileFolderModal} from '@redux/reducers/ui';
+
+import FileExplorer from '@components/atoms/FileExplorer';
+
+import {useFileExplorer} from '@hooks/useFileExplorer';
+
+import {removeIgnoredPathsFromResourceContent} from '@utils/resources';
+
+import Colors from '@styles/Colors';
+
+const {Option} = Select;
+
+const ErrorMessageLabel = styled.div`
```

Comment:
let's move all styled components to a `styled.tsx` file and then
`import * as S from "./styled` to use them here

you can also remove "Styled" prefix from the component names in this case